website/intro/use-cases.mdx

```diff
@@ -1,100 +1,92 @@
+
+For example, when a service registers with [HashiCorp Consul](https://www.consul.io/), [Consul-Terraform-Sync](https://www.consul.io/docs/nia) can automatically generate Terraform configuration to expose appropriate ports and adjust network settings for any SDN that has an associated Terraform provider. Network Infrastructure Automation (NIA) allows you to safely approve the changes that your applications require, without having to manually translate tickets from developers into the changes you think their applications need.
 
-You can use Terraform to codify the configuration for software defined networks.
-Terraform can then use this configuration to automatically set up and modify settings by interfacing with the control layer. This allows the configuration to be
-versioned and changes to be automated. For example, you can [use Terraform to configure AWS VPC](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc).
+### Resources
 
-## Resource Schedulers
+- Try the [Network Infrastructure Automation with Consul-Terraform-Sync Intro](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-intro?in=consul/network-infrastructure-automation) tutorial on Hashicorp Learn to install Consul-Terraform-Sync on a node. You will then configure it to communicate with a Consul datacenter, react to service changes, and execute an example task.
+- Try the [Consul-Terraform-Sync and Terraform Enterprise/Cloud Integration](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-terraform-enterprise?in=consul/network-infrastructure-automation) tutorial on HashiCorp Learn to configure Consul-Terraform-Sync to interact with Terraform Enterprise and Terraform Cloud.
 
-In large-scale infrastructures, static assignment of applications to machines
-becomes increasingly challenging. To solve that problem, there are a number
-of schedulers like Borg, Mesos, YARN, and Kubernetes. These can be used to
-dynamically schedule Docker containers, Hadoop, Spark, and many other software
-tools.
 
-Terraform is not limited to physical providers like AWS. Resource schedulers
-can be treated as a provider, enabling Terraform to request resources from them.
-This allows Terraform to be used in layers: to setup the physical infrastructure
-running the schedulers as well as provisioning onto the scheduled grid.
+## Kubernetes
+
+Kubernetes is an open-source workload scheduler for containerized applications. Terraform lets you both deploy a Kubernetes cluster and manage its resources (e.g., pods, deployments, services, etc.). You can also use the [Kubernetes Operator for Terraform](https://github.com/hashicorp/terraform-k8s) to manage cloud and on-prem infrastructure through a Kubernetes custom resource definition (CRD) and Terraform Cloud.
```

Comment:
```suggestion
Kubernetes is an open source workload scheduler for containerized applications. Terraform lets you both deploy a Kubernetes cluster and manage its resources (e.g., pods, deployments, services, etc.). You can also use the [Kubernetes Operator for Terraform](https://github.com/hashicorp/terraform-k8s) to manage cloud and on-prem infrastructure through a Kubernetes Custom Resource Definition (CRD) and Terraform Cloud.
```

Elsewhere we say "open source" instead.

Reply:
ooooo good to know!  and thank you - I notice that me not capitalizing these acronyms is a big problem of mine....